jwst/background/background_sub.py

```diff
@@ -90,6 +90,9 @@ def average_background(bkg_list, sigma, maxiters):
     cdata = np.zeros(((num_bkg,) + image_shape))
     cerr = cdata.copy()
 
+    if bkg_dim == 3:
+        accum_dq_arr = np.zeros((image_shape), dtype=np.int32)
```

Comment:
Might want to use *u*int32 for the data type, for consistency with the DQ arrays in the models.